config/schedule.rb

```diff
@@ -3,19 +3,32 @@
 set :output, 'log/cdx_index_rollup.log'
 set :environment_variable, 'ROBOT_ENVIRONMENT'
 
+# These define jobs that checkin with Honeybadger.
+# If changing the schedule of one of these jobs, also update at https://app.honeybadger.io/projects/51141/check_ins
+job_type :rake_hb,
+         'cd :path && :environment_variable=:environment bundle exec rake --silent ":task" :output && curl --silent https://api.honeybadger.io/v1/check_in/:check_in'
+
 every 1.day, roles: [:rollup] do
-  rake 'cdxj:index:rollup:level1'
+  set :check_in, Settings.honeybadger_checkins.rollup_level1
+  rake_hb 'cdxj:index:rollup:level1'
+end
+
+every 1.week, roles: [:rollup] do
+  set :check_in, Settings.honeybadger_checkins.cleanup_indexes
+  rake_hb 'cdxj:index:cleanup:indexes'
 end
 
 every 1.week, roles: [:rollup] do
-  rake 'cdxj:index:cleanup:indexes'
-  rake 'cdxj:index:cleanup:empty_directories'
+  set :check_in, Settings.honeybadger_checkins.cleanup_empty_directories
+  rake_hb 'cdxj:index:cleanup:empty_directories'
 end
 
```

Comment:
this block got split into two (it used to set two different rake tasks on the same schedule, but they actually run separately in cron and should check-in separately with their own keys)